network/drivers/bridge.md

```diff
@@ -181,18 +215,18 @@ By default, traffic from containers connected to the default bridge network is
 two settings. These are not Docker commands and they affect the Docker host's
 kernel.
 
-1.  Configure the Linux kernel to allow IP forwarding.
+1. Configure the Linux kernel to allow IP forwarding.
```

Comment:
Hm these instructions look really outdated. These were introduced ~9 years ago in https://github.com/docker/docs/commit/4ab18fa2ac119c26516f2156dabda9dd0a781301, which is a "refactoring" of https://github.com/docker/docs/commit/ca9c35cdf772ddb121447b3877dbcf8caa4c5cdb.

This whole section can be deleted :wink: